src/org/daisy/dotify/formatter/impl/search/CrossReferenceHandler.java

```diff
@@ -39,8 +39,8 @@
     private static final String PAGE_NUMBER_OF_FIRST_CONTENT_PAGE_OF_VOLUME =
             "page-number-of-first-content-page-of-volume-";
     private Set<String> pageIds;
-    private boolean overheadDirty = false;
     private boolean readOnly = false;
+	private static final Overhead INITIAL_OVERHEAD = new Overhead(0, 0);
```

Comment:
Have we run `gradle check` on this PR? This seems miss-aligned. And I'm a bit curious why the automatic tests weren't trigged but that is a different issue.

Reply:
Yes I should still do that.

Reply:
Fixed